src/manifest/mod.ts

```diff
@@ -286,7 +286,10 @@ export class SlackManifest {
     }
 
     // External Auth providers
-    if (def.externalAuthProviders) {
+    if (
+      def.externalAuthProviders && Array.isArray(def.externalAuthProviders) &&
+      def.externalAuthProviders.length > 0
+    ) {
```

Comment:
```ts
if (def.externalAuthProviders?.length) { ... }
```

Could we just get away with the above? TS should ensure that the value passed is an array, then if `def.externalAuthProviders` doesn't exist or has a length of 0 it'll be falsy

Reply:
deno-ts still complains about it being possibly undefined but I can bring it down to:

```
if (
      def.externalAuthProviders &&
      def.externalAuthProviders?.length > 0
    ) {
```